examples/firmware_drivers.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+)
+
+func main() {
+	var (
+		ClientOV *ov.OVClient
+	)
+	apiversion, _ := strconv.Atoi(os.Getenv("ONEVIEW_APIVERSION"))
+	ovc := ClientOV.NewOVClient(
+		os.Getenv("ONEVIEW_OV_USER"),
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		apiversion,
+		"*")
+
+	// Get all the Firmware Baseline available
+	firmware, err := ovc.GetFirmwareBaselineList("", "", "")
+	if err != nil {
+		panic(err)
+	} else {
+		fmt.Println("#-----Got Firmware Baseline-----#")
+		for i := range firmware.Members {
+			jsonResponse, _ := json.MarshalIndent(firmware.Members[i], "", "  ")
+			fmt.Print(string(jsonResponse), "\n\n")
+		}
+	}
+
+	id := strings.Split(firmware.Members[0].Uri.String(), "/")[3] //eg: Synergy_Custom_SPP_2021_02_01_Z7550-97110
+	// Get Firmware Baseline by id
+	firmware2, err := ovc.GetFirmwareBaselineByUri(id)
```

Comment:
Can you rename this func to GetById instead of URI since we are passing id as param